src/main/java/com/example/auctionapp/seeder/DatabaseSeeder.java

```diff
@@ -273,8 +282,8 @@ private void seedProductTable() {
                             "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
                             1000.,
                             subcategories.get(2),
-                            LocalDateTime.now().minusHours(77),
-                            LocalDateTime.now().plusHours(3),
+                            LocalDateTime.now(ZoneOffset.UTC).minusHours(77),
+                            LocalDateTime.now(ZoneOffset.UTC).plusHours(3),
```

Comment:
Same, we can have some kind of database seeder utility with static methods that return example products.
Some of the properties will be the same, like the description, time etc, while some can be passed as arguments.

Reply:
I did not do this, because only property that is same across all products is description. All others need to be different because of testing different functionalities (new arrivals, last chance, feature etc logic depend on these properties). At least I put description in a separate variable.